src/main/java/edu/cornell/kfs/sys/util/CuSqlChunk.java

```diff
@@ -15,19 +15,30 @@ public final class CuSqlChunk implements CharSequence {
 
     private StringBuilder queryString;
     private Stream.Builder<SqlParameterValue> parameters;
+    private CharSequence underlyingCharSequence;
```

Comment:
Why is this needed?  I see this is set to queryString in the constructor, if something happens where queryString is set to a new object, the two might get out of synch

Reply:
This is part of the cleanup I had hinted at in one of my earlier comments. I need the `CuSqlChunk` class to implement `CharSequence` because of how this SQL-building API has been configured, but I also want to clear out the underlying string content when the object is no longer needed. This PR will use this new field to handle the implemented `CharSequence` methods; it's initially set to the underlying `StringBuilder` but it will be replaced with an empty string after this object has been used up. (Prior to the change, this class would replace the `StringBuilder` with a fresh empty one after the object gets used up. I was trying to structure this to avoid the creation of new unnecessary `StringBuilder` objects.)

I could replace this with some other setup, such as having some of the relevant `CharSequence` methods throw exceptions if the object has already been used up. Thoughts?

Reply:
I am not sure what the best action is right now.  Having both queryString and underlyingCharSequence seems to add some confusion that I would like to avoid if possible.

Reply:
Ok, I'll refactor this to remove `underlyingCharSequence`.